examples/disc_brake/read_disc_brake_abaqus_data.py

```diff
@@ -0,0 +1,53 @@
+import os
+import logging
+import numpy as np
+from aphin.utils.data import DiscBrakeDataset
+from aphin.utils.configuration import Configuration
+
+
+def main():
+    """
+    Following the workflow to obtain simulation data from Abaqus, see https://doi.org/10.18419/darus-4418
+    We obtain the data in form of .txt files
+    This function reads and processes the .txt files.
+    The data is saved into a more convenient (faster reading, less storage) .npz file.
+    Hence, this preprocessing function needs to be executed only once or if changes in the Abaqus model were conducted.
+
+    Please define the path to the folder with the .txt files below.
+    """
+    # -----------------------------------------
+    # USER INPUT REQUIRED
+    # -----------------------------------------
+    disc_brake_txt_path = "/path/to/txt/folder/"
```

Comment:
If you use a main function I'd put the txt_path as function argument and also rename the function to a more explanatory name

Reply:
Changed :)